packages/trader/src/Assets/Trading/Categories/trade-categories.tsx

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
 import { Text } from '@deriv/components';
+import { getLocalizedBasis } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
 import AccumulatorTradeDescription from './accumulator-trade-description';
 import { TurbosTradeDescription } from './turbos-trade-description';
 
 // Templates are from Binary 1.0, it should be checked if they need change or not and add all of trade types
 // TODO: refactor the rest of descriptions to use them as components like AccumulatorTradeDescription
-const TradeCategories = ({ category, onClick }: { category?: string; onClick: () => void }) => {
+const TradeCategories = ({
```

Comment:
better extract this out to types instead, props are getting long

Reply:
Sure, we can do this